### PR TITLE
Add conda package definition

### LINKED ISSUE #73: Publish as a conda package
**Is your feature request related to a problem? Please describe.**
Some of the dependencies are much easier to install using conda than using pip. We also recommend that users install MapReader in a conda environment. Mixing conda installs and pip installs is known to be problematic.

**Describe the solution you'd like**
Publish MapReader as a conda package, eliminating the need to mix conda installs and pip installs.

### PR DESCRIPTION
**Pull request by andrewphilipsmith**
- [x] Check install in conda 
- [x] Check dependency resolution
- [x] Check and document impact on #62 
- [x] Check and document impact on #64 
- [x] Setup GH Action to push to conda-forge
- [x] Check if this makes #47 redundent

Will close #73

### CODE REVIEW

**Comment by andrewphilipsmith** [comment]
I believe that this is now ready to review/merge.

There are a couple of things that are not adequately addressed in this PR. I have opened separate issues for these:
- https://github.com/Living-with-machines/MapReader/issues/92
- https://github.com/Living-with-machines/MapReader/issues/93

I think that it would be best to merge this PR and deal with those two issues as future PRs.

**Comment by andrewphilipsmith** [comment]
@rwood-97 I believe that this is now ready to review/merge.

There are a couple of things that are not adequately addressed in this PR. I have opened separate issues for these:
- https://github.com/Living-with-machines/MapReader/issues/92
- https://github.com/Living-with-machines/MapReader/issues/93

I think that it would be best to merge this PR and deal with those two issues as future PRs.

File: setup.py [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+import versioneer
 
 EXTRAS_REQUIRE = {
         'geo': [
             "geopy==2.1.0",
             "rasterio>=1.2.10,<2.0.0",
             "keplergl>=0.3.2,<0.4.0",
             "simplekml>=1.3.6,<2.0.0"
-            ]
-        }
```
[the remark this answers is not in the document]
Comment (andrewphilipsmith) [reply setup.py:10]: Good point - I had started with the "basic" install and had forgotten to come back to the geo bits.

[the remark this answers is not in the document]
Comment (andrewphilipsmith) [reply setup.py]: The geo packages are now included in the conda package.

File: .github/workflows/publish-to-conda-forge.yml
Comment (rwood-97) [inline .github/workflows/publish-to-conda-forge.yml:1]: We should probably have equivalent of this for PyPI

File: mapreader/download/tileserver_access.py
Comment (rwood-97) [inline mapreader/download/tileserver_access.py:1]: Can you quickly merge main again to fix merge conflicts with these? (this also fixes the issue with docs not building)

File: mapreader/download/tileserver_scraper.py
Comment (rwood-97) [inline mapreader/download/tileserver_scraper.py:1]: As above

File: .github/workflows/publish-to-conda-forge.yml
Comment (andrewphilipsmith) [reply .github/workflows/publish-to-conda-forge.yml:1]: See #47

File: mapreader/download/tileserver_access.py
Comment (andrewphilipsmith) [reply mapreader/download/tileserver_access.py:1]: Done

File: mapreader/download/tileserver_scraper.py
Comment (andrewphilipsmith) [reply mapreader/download/tileserver_scraper.py:1]: Done

**Pull request merged**